meteor-app/imports/ui/service/service-context.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, createContext } from 'react'
+import ServiceItems from '/imports/api/assessments/serviceItems'
+import Services from '/imports/api/assessments/services'
+// import service from './index'
+
+console.log(ServiceItems)
+
+// import {createContainer} from "meteor/react-meteor-data"
+
+export const ServiceContext = createContext()
+
+const data = [
+  { title: 'item1', isMajMin: false },
```

Comment:
Please remove this sample data from this file.

Reply:
it's removed by Murtaza I think